code/datums/materials/basemats.dm

```diff
@@ -189,6 +189,29 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	strength_modifier = 0.8
 	value_per_unit = 0.025
 
+/datum/material/wood
+	name = "wood"
+	id = "wood"
+	desc = "Flexible, durable, but flamable. Hard to come across in space."
+	color = "#bb8e53"
+	strength_modifier = 0.5
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	value_per_unit = 0.06
+	beauty_modifier = 0.1
+	armor_modifiers = list("melee" = 1.1, "bullet" = 1.1, "laser" = 0.4, "energy" = 0.4, "bomb" = 1, "bio" = 0.2, "rad" = 0, "fire" = 0, "acid" = 0.3)
+
+/datum/material/wood/on_applied_obj(atom/source, amount, material_flags)
+	. = ..()
+	if(istype(source, /obj/item))
+		var/obj/item/wooden = source
+		wooden.resistance_flags |= FLAMMABLE
```

Comment:
this should have the stat flag requirement too.

Reply:
Run this one by me one more time when you have a second, I haven't worked with bitflags before this one.

Reply:
https://github.com/tgstation/tgstation/blob/e87033d792cf4834f4fc18b0e6a2c2057859ce89/code/datums/materials/_material.dm#L50-L55